types/setup.d.ts

```diff
@@ -3,7 +3,8 @@
 
 export interface SetupOptions {
     /**
-     * PREVIEW: Stream http requests and responses instead of loading entire body in memory
+     * PREVIEW: Stream http requests and responses instead of loading entire body in memory.
+     * [Learn more here](https://aka.ms/AzFuncNodeHttpStreams)
```

Comment:
Since this isn't a markdown file, should we skip the link formatting?
```
Learn more here: https://aka.ms/AzFuncNodeHttpStreams
```

Reply:
<img width="928" alt="Screenshot 2024-02-16 at 3 41 11 PM" src="https://github.com/Azure/azure-functions-nodejs-library/assets/11282622/edc0e17d-343f-4b7b-99b5-df195c05f3bc">

VS Code actually supports markdown links for their intellisense, which is why I do it. Probably not every tool supports it and obviously someone could look at the types file directly, but I feel like even with the braces/parentheses it's still pretty readable, but let me know if you disagree